docs/azure_jumpstart_ag/contoso_supermarket/adx_dashboards/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+type: docs
+weight: 100
+toc_hide: true
+---
+
+# Contoso Supermarket data pipeline and reporting across cloud and edge - Store orders
+
+If you are on this page, by now you may have the familiarity of Contoso Supermarket, its objectives, and goals. As part of the series of various technology stack demonstration scenarios, this scenario covers the data pipeline and reporting across various Contoso Supermarket stores and services to showcase how the Point of Sale (PoS) order data flows to Azure Data Explorer (ADX) database and generate near realtime reports. Contoso management can leverage these reports to adjust their inventory and supply chain based on the product demand from customer orders across stores, day, week, month, and year. This helps optimize Contoso resources, store capacity, and saves significant cost and at the same time improves customer satisfaction and trust.
```

Comment:
Always start with a problem statement. "Contoso need to achieve X..."


One of Contoso's biggest objectives is how to use the data coming from their stores and visualize it for business intelligence by leveraging the power of the cloud.

In this scenario, Contoso wants to use their data pipeline so customer orders placed in the Point of Sale (PoS) application on the various supermarket stores, flow to [Azure Data Explorer (ADX)](https://learn.microsoft.com/azure/data-explorer/data-explorer-overview) database and generate near real-time reports. By doing so, Contoso management can leverage these reports to adjust their inventory and supply chain based on the product demand from customer orders across multiple filters - stores, day, week, month, and year. This helps optimize Contoso resources, stores supplies, saves significant costs and at the same time improves customer satisfaction and trust.

Reply:
Updated as per the comments